backend/src/main/resources/application.yaml

```diff
@@ -8,8 +8,9 @@ spring:
     password: api123
     url: jdbc:postgresql://localhost:${SR_DB_PORT:5432}/simple_report
     hikari:
-      maximum-pool-size: 20
-      max-lifetime: 900000 # Maximum lifetime for a connection to be retained in the pool, in milliseconds, once it is closed.
+      maximum-pool-size: 50 # Maximum size that the pool is allowed to reach, including both idle and active connections. Effectively reflects number of concurrent database request threads. (Ex: A size of 50 represents a maximum capacity of 50 concurrent DB requests.)
+      max-lifetime: 600000 # Maximum lifetime for a connection to be retained in the pool, in milliseconds, once it is closed.
+      minimum-idle: 30 # When a connection has been idle for 10 minutes (unless idleTimeout is overridden), it will be purged from the pool. This will allow for any un-closed, inactive transactions to be cleaned up more readily.
```

Comment:
This setting doesn't match the comment - did you mean `idleTimeout`?

Reply:
Good question! The comment is correct. `idleTimeout` sets the value at which connections will begin to be purged down to the value for `minimumIdle`. `idleTimeout` defaults to 10 minutes; if you want idle connections to be purged more quickly, you can add that setting as a separate line to override.

Reply:
I've cleaned this documentation up to address the uncertainty.